### PR TITLE
Use System.Security.Policy.Publisher from corefx (they essentially stubbed it out).

### PR DESCRIPTION
**Pull request by baulig**
Fixes the XUnit 2.4 problem.

### CODE REVIEW

**Comment by alexischr** [comment]
@monojenkins commit apidiff

**Comment by alexischr** [comment]
@monojenkins commit csproj

File: mcs/class/corlib/System.Security.Policy/Publisher.cs [hunk 1]
```diff
@@ -38,7 +38,8 @@ namespace System.Security.Policy {
 	public sealed class Publisher :
 		EvidenceBase,
 		IIdentityPermissionFactory, IBuiltInEvidence {
-	
+
+		[NonSerialized]
```
Comment (marek-safar) [inline mcs/class/corlib/System.Security.Policy/Publisher.cs]: Why did you change file which is no longer used?

Comment (baulig) [reply mcs/class/corlib/System.Security.Policy/Publisher.cs]: Left-over from testing, before replacing the file.

Comment (marek-safar) [reply mcs/class/corlib/System.Security.Policy/Publisher.cs]: Could you delete the no longer used files
